### DISCUSSION TITLE
Flux nf4 model pulled from networks tab wont load without huggingface token?

### DISCUSSION THREAD

**Discussion by cosmiconions** [vladmandic/sdnext#3400] [Q&A] [answered]
Heya everyone! I'm trying to give the Flux nf4 version a go, but when loading the model to generate an image, it throws an error that states: 
"Diffusers Failed loading model" (path to diffusers folder) "black-forest-labs/flux.1-dev is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models' If this is a private repository, make sure to pass a token having permission to this repo with `token` or log in with `huggingface-cli login`."
I can confirm it has made a local folder on my system, im not sure why it's asking for a huggingface token after it was able to be installed and such. Is adding a token really necessary to run this model?

**Comment by vladmandic** [comment, marked as answer]
its been fixed on dev branch, service release not far out.

**Reply by cosmiconions** [reply]
Awesome thanks for the reply! I knew the dev model was private, but was confused why this one appeared to be! I'll wait for the new release and give it another shot!
